smack-core/src/main/java/org/jivesoftware/smack/packet/AbstractError.java

```diff
@@ -70,6 +70,9 @@ public String getDescriptiveText() {
             descriptiveText = getDescriptiveText("en");
             if (descriptiveText == null) {
                 descriptiveText = getDescriptiveText("");
+                if (descriptiveText == null) {
+                    descriptiveText = getDescriptiveText(null);
```

Comment:
I wonder if it wouldn't be better if we *never* used `null` as key, and instead use the empty string. Then this 3 LOCs could be removed.